README.md

```diff
@@ -1,85 +1,100 @@
 # Welcome to RskJ Powpeg Node
 
+
+
 ## About
 
-Powpeg node is a specialized rskj node which interacts with both RSK and Bitcoin.
-This node is used by RSK PowPeg signatories to interact with the Bridge contract and to broadcast peg-out transactions to Bitcoin.
+Powpeg node is a specialized rskj node which interacts with both Rootstock and Bitcoin.
+This node is used by Rootstock PowPeg signatories to interact with the Bridge contract and to broadcast peg-out transactions to Bitcoin.
+
 
 ## Software Requirements
+
 1. Java JDK 1.8
-2. Bitcoin Core daemon (bitcoind) 0.17v or 0.18v 
+2. Bitcoin Core daemon (bitcoind) 0.17v or 0.18v
+3. A Java compatible IDE. Recommended [IntelliJ IDEA](https://www.jetbrains.com/idea/download) as this guide covers the setup with it
 
-## Setting up the project
 
-### Required configurations
+**Not sure how to install any of these? See [software installation help](#software-installation-help)**
+
+
+## Software installation help
+Disclaimer: this documentation will be specific for macOS operative system.
 
-1. You will need a private key file to be used by the pegnatory to sign BTC/RSK transactions. Follow these steps to generate it.
+### **Java JDK 1.8**
 
-- Using **GenNodeKeyId** class in the rskj project
+Although optional we recommend to install jenv to manage different Java versions, to do that run: `brew install jenv`
 
-- Set String generator = "federator1";
+Add jenv to your path as it says on the output of jenv brew installation.
 
-- Run the Class to generate a private key
+Download [oracle-jdk8](https://www.oracle.com/ar/java/technologies/javase/javase8-archive-downloads.html) and install it.
```

Comment:
we recommended zulu first, but Jose told us it was better to recommend to
download the official java version from Oracle

Reply:
Well, the thing is Oracle JDK license is not free and open source(AFAIK), it is limited only to personal use. On the other hand, OpenJDK is free and open source and not limited to personal use. That's my concern. Taking into account that RSK is an open-source project, recommend this instead of the Oracle JDK I think makes more sense.

Reply:
i think you are right. maybe we can leave both